plugins/modules/openssl_pkcs12.py

```diff
@@ -302,6 +317,18 @@
 else:
     CRYPTOGRAPHY_FOUND = True
 
+CRYPTOGRAPHY_COMPATIBILITY2022_ERR = None
```

Comment:
@felixfontein Thoughts on initializing as `False` so the variable comes into existence as a `bool` and remains as such?

Reply:
Addressed on Matrix, `that variable is either `None` or a string (traceback of the error)`.